v03_pipeline/lib/tasks/reference_data/update_variant_annotations_table_with_updated_reference_dataset_test.py

```diff
@@ -974,152 +537,144 @@ def test_update_vat_with_updated_rdc_mito_38(
+                            clinvar=ReferenceDataset.clinvar.enum_globals,
+                            dbnsfp=ReferenceDataset.dbnsfp.enum_globals,
+                            sorted_transcript_consequences=hl.Struct(
+                                biotype=BIOTYPES,
+                                consequence_term=TRANSCRIPT_CONSEQUENCE_TERMS,
+                                lof_filter=LOF_FILTERS,
+                            ),
+                            mitotip=hl.Struct(
+                                trna_prediction=MITOTIP_PATHOGENICITIES,
+                            ),
                         ),
-                        high_constraint_region_mito=hl.Struct(),
-                        local_constraint_mito=hl.Struct(),
-                        sorted_transcript_consequences=hl.Struct(
-                            biotype=BIOTYPES,
-                            consequence_term=TRANSCRIPT_CONSEQUENCE_TERMS,
-                            lof_filter=LOF_FILTERS,
+                        migrations=[],
+                        updates=set(),
+                    ),
+                ],
+            )
+            self.assertCountEqual(
+                ht.collect(),
+                [
+                    hl.Struct(
+                        locus=hl.Locus(
+                            contig='chrM',
+                            position=1,
+                            reference_genome='GRCh38',
                         ),
-                        mitotip=hl.Struct(
-                            trna_prediction=MITOTIP_PATHOGENICITIES,
+                        alleles=['A', 'C'],
+                        helix_mito=hl.Struct(
+                            AC_het=0,
+                            AF_het=0.0,
+                            AN=195982,
+                            max_hl=None,
+                            AC_hom=0,
+                            AF_hom=0,
+                        ),
+                        hmtvar=hl.Struct(score=0.6700000166893005),
+                        mitimpact=hl.Struct(score=0.42500001192092896),
+                        mitomap=hl.Struct(pathogenic=None),
+                        gnomad_mito=hl.Struct(
+                            AC_het=0,
+                            AF_het=0.0,
+                            AN=195982,
+                            max_hl=None,
+                            AC_hom=0,
+                            AF_hom=0,
+                        ),
+                        local_constraint_mito=hl.Struct(score=0.5),
+                        clinvar=hl.Struct(
+                            alleleId=None,
+                            conflictingPathogenicities=None,
+                            goldStars=None,
+                            pathogenicity_id=None,
+                            assertion_ids=None,
+                            submitters=None,
+                            conditions=None,
+                        ),
+                        dbnsfp=hl.Struct(
```

Comment:
`dbnsfp_mito` has fewer fields than the new mito `dbnsfp` annotation since we used to treat it as a separate dataset, so we'll need to decide how to handle this (keep the extra fields or add a config somewhere to filter down to the fields we want for the mito dataset type)